gui/src/callbacks.cc

```diff
@@ -438,6 +441,44 @@ void find_or_create_scroom(PresentationInterface::Ptr presentation)
   create_scroom(presentation);
 }
 
+void
+onDragDataReceived(GtkWidget *, GdkDragContext *, int, int,
+                        GtkSelectionData *seldata, guint, guint,
+                        gpointer)
+{
+  printf("Dropping file(s) onto SCROOM %s\n", (gchar*)seldata->data);
```

Comment:
I try to avoid using all caps. So I'd prefer `scroom`, though `Scroom` would be fine too.